doc/source/project/organization.rst

```diff
@@ -175,8 +175,23 @@ The Galaxy *committers* group may only commit directly to Galaxy (i.e. outside
 of a pull request and not following the procedures described here) the
 following two categories of patches:
```

Comment:
Oops.  This is what i get for screwing up the original commit and re-applying it quickly :)   Good catch.